src/main/scala/stores/store/valueobjects/ItemsRowId.scala

```diff
@@ -7,9 +7,26 @@
 package io.github.pervasivecats
 package stores.store.valueobjects
 
-import stores.Id
+import eu.timepit.refined.api.RefType.applyRef
+
+import stores.{Id, Validated, ValidationError}
 
 trait ItemsRowId {
 
   val value: Id
 }
+
+object ItemsRowId {
+
+  final private case class ItemsRowIdImpl(value: Id) extends ItemsRowId
+
+  case object WrongItemsRowId extends ValidationError {
+
+    override val message: String = "The items row id is a negative value"
```

Comment:
Choose between "invalid" or "negative value" between all value objects. The second is preferred